src/js/ui/function.js

```diff
@@ -0,0 +1,92 @@
+// eslint-disable-next-line no-unused-vars
+import { VENDING_MACHINE_CONSTANT } from '../service/constant.js';
+import { ELEMENT, querySelector } from './element.js';
+
+const inputName = querySelector(ELEMENT.INPUT.PRODUCT_NAME);
+const inputPrice = querySelector(ELEMENT.INPUT.PRODUCT_PRICE);
+const inputAmount = querySelector(ELEMENT.INPUT.PRODUCT_AMOUNT);
+const inputChargeAmount = querySelector(ELEMENT.INPUT.CHARGE_AMOUNT);
```

Comment:
보통 element를 표현하는 변수는 이런식으로 쓰인답니다!
```suggestion
const $inputName = querySelector(ELEMENT.INPUT.PRODUCT_NAME);
const $inputPrice = querySelector(ELEMENT.INPUT.PRODUCT_PRICE);
const $inputAmount = querySelector(ELEMENT.INPUT.PRODUCT_AMOUNT);
const $inputChargeAmount = querySelector(ELEMENT.INPUT.CHARGE_AMOUNT);
```

혹은 이렇게 namespace로 묶어서 표현할 수도 있겠네요.
```suggestion
const $elements = {
  inputName: querySelector(ELEMENT.INPUT.PRODUCT_NAME),
  inputPrice: querySelector(ELEMENT.INPUT.PRODUCT_PRICE),
  inputAmount: querySelector(ELEMENT.INPUT.PRODUCT_AMOUNT),
  inputChargeAmount: querySelector(ELEMENT.INPUT.CHARGE_AMOUNT),
};
```